src/backend/d3d12/FramebufferD3D12.h

```diff
@@ -41,6 +43,9 @@ namespace d3d12 {
             Device* device;
             DescriptorHeapHandle rtvHeap;
```

Comment:
nit: these should be null initialized so that they are nullptr if there are no color / DS attachments, which would make crashes easier to understand.